shared/src/main/scala/mlscript/helpers.scala

```diff
@@ -472,20 +473,28 @@ trait NuDeclImpl extends Located { self: NuDecl =>
 }
 
 trait TypingUnitImpl extends Located { self: TypingUnit =>
-  def showDbg: Str = entities.map {
-    case t: Term => t.toString
+  def showDbg: Str = entities.iterator.map {
+    case t: Term => t.print(false)
+    case d: NuDecl => d.showDbg
+    case c: Constructor => c.showDbg
+    case e => lastWords(s"Unexpected typing unit entity: $e")
+  }.mkString("‹", "; ", "›")
+
+  def showDbgTop: Str = entities.iterator.map {
+    case t: Term => t.print(false)
     case d: NuDecl => d.showDbg
-    case c: Constructor => c.toString
+    case c: Constructor => c.showDbg
     case e => lastWords(s"Unexpected typing unit entity: $e")
   }.mkString("{", "; ", "}")
-  override def toString: String = s"‹${entities.mkString("; ")}›"
+
```

Comment:
Regarding this, the pretty printing for `TypingUnit` used two types of brackets: `‹›` and `{}`, in order not to have a lot of irrelevant changes in DiffTests, I kept the two print outputs and named the version using `{}` as `showDbgTop`. I want to ask if we can only keep one version of `showDbg` function.

Reply:
Would just using `‹›`  create too many changes? Anyway, it's better for it to be consistent, so let's use it.

Reply:
Yes, it changes 55 files. See https://github.com/chengluyu/mlscript/commit/e4e03b1a0578e02e03072b281f61d92cea724a62 for details.

Reply:
How many files does picking the other one create? :P

Reply:
> How many changes does picking the other one create? :P

```
M  compiler/shared/test/diff/Lifter.mls
M  compiler/shared/test/diff/LifterBlks.mls
M  shared/src/main/scala/mlscript/helpers.scala
M  shared/src/test/diff/parser/BasicSyntax.mls
M  shared/src/test/diff/parser/New.mls
M  shared/src/test/scala/mlscript/DiffTests.scala
```

Reply:
So, let's just use `{}`? 🫢

Reply:
Sure.